qt/lc_qpreferencesdialog.cpp

```diff
@@ -78,6 +78,7 @@ lcQPreferencesDialog::lcQPreferencesDialog(QWidget* Parent, lcPreferencesDialogO
 	else
 		ui->antiAliasingSamples->setCurrentIndex(0);
 	ui->edgeLines->setChecked(mOptions->Preferences.mDrawEdgeLines);
+	ui->LegoStyleDisplay->setChecked(mOptions->LegoStyleDisplay);
```

Comment:
Wouldn't it make more sense to merge this option with the stud logos and call it "Stud Style"?

Reply:
It would also make the rest of the code simpler

Reply:
That change does make sense and I did think about it.

My thinking to go with the current config was that the Lego Style also include edge line colour updates. Moreover, it could be possible in the future to enable Lego Style Display with Stud Option 1. Anyway, if you prefer merging the two, I’ll do it. It can always be reconfigured for future updates if needed.

Cheers,